code/modules/mob/living/carbon/examine.dm

```diff
@@ -50,9 +50,9 @@
 			else
 				msg += "[t_He] [t_has] minor bruising.\n"
 		else
-			if(prob(33))
+			if(prob(50))
```

Comment:
I mean the first if is prob(33) the second should be prob(50)

Think of it this way, you start with 100 cases

You take 33% of that, you now have 66 cases

Take another 33% of that (22) and you'll have 44 cases

Taking 50% of the 66 cases is what you want in the second if

TL;DR

if(prob(50))
   //50% chance
else if(prob(50))
   //25% chance
else
   //25% chance

if(prob(33))
  //33% chance
else if(prob(33))
  //22% chance
else
  //44% chance

if(prob(33))
  //33% chance
else if(prob(50)
  //33% chance
else
  //33% chance